clients/algoliasearch-client-scala/build.sbt

```
@@ -0,0 +1,53 @@
+organization := "com.algolia"
+name := "algoliasearch-scala"
+description := "Scala client for Algolia Search API"
```

```suggestion
description := "⚡️ A fully-featured and blazing-fast Scala API client to interact with Algolia."
```

maybe? since there's other clients than search